cms/djangoapps/contentstore/tests/test_import.py

```diff
@@ -283,3 +283,54 @@ def _verify_split_test_import(self, target_course_name, source_course_name, spli
         }
 
         self.assertEqual(remapped_verticals, split_test_module.group_id_to_child)
+
+    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
+    def test_video_components_present_while_import(self, store):
```

Comment:
I don't think this test method is needed anymore.
The [test_reimport](https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/tests/test_import.py#L186) should be sufficient.